LoopKit/DosingDecisionStore.swift

```diff
@@ -283,7 +283,7 @@ extension DosingDecisionStore {
                     object.date = dosingDecisionData.date
                     object.data = dosingDecisionData.data
                 }
-                self.store.save { error = $0 }
+                error = self.store.save()
```

Comment:
I'm a bit confused... I would have expected a concomitant change to `PersistenceController.save()`.   Forgive me if I forgot...was this in another PR?

Reply:
Yes, that was changed in an earlier PR. Specifically: https://github.com/tidepool-org/LoopKit/pull/217/files#diff-0548cbd5c4f052bf87c5bf6b86bfde10R112.